openmetadata-ui/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -392,10 +396,9 @@ const DatasetDetailsPage: FunctionComponent = () => {
               setTableQueries(tableQueries);
             })
             .catch(() =>
-              showToast({
```

Comment:
@ShaileshParmar11 , Please check the API response error message here too.

Reply:
For this code is updated. but its not showing here as its commented on deleted line.